test/requests/formatting_expectations_test.rb

```diff
@@ -22,4 +22,7 @@ def assert_expectations(source, expected)
     assert_empty(stdout)
     assert_equal(expected, result)
   end
+
+  def initialize_params(_expected)
```

Comment:
Why do we need to redefine this? Shouldn't the default implementation work (or no-op)?

Reply:
I'll explore this in a follow up PR.